test/e2e/framework/util.go

```diff
@@ -876,9 +876,21 @@ func (f *Framework) MatchContainerOutput(
 	Logf("Trying to get logs from node %s pod %s container %s: %v",
 		podStatus.Spec.NodeName, podStatus.Name, containerName, err)
 
-	// Sometimes the actual containers take a second to get started, try to get logs for 60s
-	logs, err := e2epod.GetPodLogs(f.ClientSet, ns, podStatus.Name, containerName)
+	// Sometimes the actual containers take a few seconds to get started, try to get logs for 1m.
+	var logs string
+	var podLogErr error
+	err = wait.PollImmediate(
+		2*time.Second, /* from pods/wait.go:poll */
+		1*time.Minute,
+		func() (bool, error) {
+			logs, podLogErr = e2epod.GetPodLogs(f.ClientSet, ns, podStatus.Name, containerName)
+			if err == nil {
```

Comment:
Doesn't here need to check `podLogErr` as a returned value of `e2epod.GetPodLogs()` instead of `err`?

Reply:
thanks for catching this! I think I started with `err` but changed it later to `podLogErr` to join both errors.